rust+wasm/deny.toml

```diff
@@ -0,0 +1,197 @@
+# This template contains all of the possible sections and their default values
```

Comment:
What do you think about stripping the doc? It makes it hard to parse the actual config. I'm assuming the info here can be found on the doc site.

Reply:
@appcypher I thought it may be helpful for those editing it (as this is just the template). I saw that other "bigger" libs like linkered still had it, as most people run `cargo deny init` and then modify as it goes.

Reply:
fwiw - when I am spinning up a new rust project the doc speeds me up considerably - reminding myself what was what - this is why it gets generated via `cargo deny init` :)